pkgs/os-specific/linux/libratbag/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, meson, ninja, pkgconfig
+, glib, systemd, udev, libevdev, gitMinimal, check, valgrind, swig, python3 }:
+
+stdenv.mkDerivation rec {
+  name = "libratbag-${version}";
+  version = "0.9.903";
+
+  src = fetchFromGitHub {
+    owner  = "libratbag";
+    repo   = "libratbag";
+    rev    = "v${version}";
+    sha256 = "0cr5skrb7a5mgj7dkm647ib8336hb88bf11blaf6xldafi8b0jlj";
+  };
+
+  nativeBuildInputs = [
+    (python3.withPackages (ps: with ps; [ evdev pygobject3 ]))
```

Comment:
If this is moved to buildInputs, the patch shebangs for "ratbagctl" gives the wrong python, and program fails because it can't import the right modules. This seemed like the wrong place to put it based on nixos manual's description of the different build inputs, but I am not sure how to get around that.